src/main/charts/bitbucket/values.yaml

```diff
@@ -445,20 +445,27 @@ bitbucket:
     # needs extra annotations.
     #
     annotations: {}
-    
-  # Enable or disable security context in StatefulSet template spec.
-  # Enabled by default with UID 2003. Disable when deploying to OpenShift,
-  # unless anyuid policy is attached to service account.
+
+  # Standard K8s field that holds pod-level security attributes and common container settings.
   #
   securityContext:
-    
-    # -- Set to 'true' to enable the security context
-    #
-    enabled: true
-    
+
     # -- The GID used by the Bitbucket docker image
     #
-    gid: "2003"
+    fsGroup: 2003
+
+#    # -- Set to 'true' to enable the security context (Deprecated)
+#    #
+#    enabled: true
+#
+#    # -- The GID used by the Bitbucket docker image (Deprecated)
+#    #
+#    gid: "2003"
+
+
+  # -- Standard K8s field that holds security configurations that will be applied to a container.
```

Comment:
Include link to https://kubernetes.io/docs/tasks/configure-pod-container/security-context/